lib/cryptoexchange/exchanges/coss/market.rb

```diff
@@ -2,10 +2,10 @@ module Cryptoexchange::Exchanges
   module Coss
     class Market < Cryptoexchange::Models::Market
       NAME = 'coss'
-      API_URL = 'https://exchange.coss.io/api'
+      API_URL = 'https://market.coss.io/api'
 
       def self.trade_page_url(args={})
-        "https://coss.io/c/trade?s=#{args[:base].upcase}_#{args[:target].upcase}"
+        "https://market.coss.io/api/trades/#{args[:base].upcase}_#{args[:target].upcase}"
```

Comment:
this should not gives the api link. It should gives the link where user can operate the trade.

In coss's case, there's no public viewing of trade page unless you are signed-up. Also signed-up is now invitation-only. So, we return `nil`

Reply:
ah i see